### PR TITLE
DEV-490 - capacitor lib updates

### PR DESCRIPTION
**Pull request by randyhoulahan**
very late PR.   before cop the app store rejected and Christine got it in fast through a friend.  I had covid and never PR'ed it.

### CODE REVIEW

**Review by rqk2k — APPROVED** [review]

File: components/article.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    </div>
+  </template>
+  
+  <script>
+  export default {
+    name: 'ArticleHome',
+    props: ['title', 'content', 'blob']
+  }
+  function data(){
+    console.log('this.blob',this.blob)
```
Comment (blaisefonseca) [inline components/article.vue:18]:
```suggestion
```

File: components/Calender/src/components/index.vue [hunk 2]
```diff
@@ -148,8 +148,8 @@ function setQueryString(interations){
 
   if(interations) this.$router.replace(route)
 
-  const start = aDateTime.toUTC().toISO()
-  const end   = endDateTime.toUTC().toISO()
+  const start = aDateTime.toISO({includeOffset:false})
+  const end   = endDateTime.toISO({includeOffset:false})
```
Comment (sbilodeau) [inline components/Calender/src/components/index.vue:152]: this option is to remove the offet indicator of ta date. 

I think it's an incorrect way that either fix/cause the entries that disapear on edge case (begining/end of week entries) we experience it in Nairobi that was ok in montreal `UTC-5` vs `UTC+2`. Clealy a timezone issue that the application does not seems to handle 

```javascript
{includeOffset:false}
```

File: components/Calender/src/components/index.vue [hunk 3]
```diff
@@ -174,14 +174,16 @@ function mapByDay(events){
     const   dayStart         = DateTime.fromISO(raw[i].start).startOf('day')
     const   dayEnd           = DateTime.fromISO(raw[i].start).endOf('day')
     const   start            = DateTime.fromISO(raw[i].start)
-    const   dayStartText     = dayStart.toISODate()
+    const   dayStartText     = dayStart.toISODate({includeOffset:false})
```
Comment (sbilodeau) [inline components/Calender/src/components/index.vue:177]: same as above

**Review by sbilodeau — CHANGES_REQUESTED** [review]

**Comment by sbilodeau** [comment]
the PR is large and mixing multiple features in one PR.
- upgrade of library
- fix of date edgecase
- article stuff
- etc
it very important to try to make mutiple PRs. one for each feature.

**Comment by blaisefonseca** [comment]
@rqk2k I think you should create a separate PR for your changes. this is mixing two things

**Pull request closed without merging**